css-two-column-layout/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <h1>Scale, Speed, Agility</h1>
+          <p>
+            We've dedicated thousands of man hours to making healthcare
+            institutions in the Helium network run 10x more efficiently
+            and deliver an unparalleled quality of care.
+          </p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-half">
+        <img src="images/software.svg" alt="software-icon">
+          <h3>Intuitive, <br> Smart Software</h3>
+            <p class="body">
+              We build products that transform how healthcare facilities
+              run across Africa - making them radically more efficient
+              operationally and financially, a complete turnaround.
+            </p>
+      </div>
+      <div class="column-half">
+        <img src="images/practice.svg" alt="medical-icon">
+          <h3>Practice Medicine, Not Administration</h3>
+            <p class="body">
+              Our all-in-one electronic health records (EHR), hospital
+              management (HMIS) &amp; billing system helps streamline your facility,
+              reduce waste and improve quality of care.
+            </p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-half">
+        <img src="images/circumstance.svg" alt="circumstance-icon">
+          <h3>For Any Circumstance</h3>
+            <p class="body">
+              Our intuitive system is built around the peculiarities of your facility;
+              we support offline deployments, sync to the cloud and run on any device
+              24/7.
+            </p>
```

Comment:
Your indentation here is a bit off. Remember, sibling element should always be at the same level of indentation.